release_notes.md

```diff
@@ -3,56 +3,71 @@
+  - These fields allow analysts to trace exactly which commit was last applied and whether it succeeded, making CR tracking more reliable.
 
-    - Apply Latest Content
+- **Task Management Improvements**
 
-    - Save Production Settings
+  - Every task now includes:
 
-    - Map Source Control Username
+    - A **Reset Task** button to reset a stuck task (anything not in *Pending*).
+    - An **Abort** button to stop a task in progress.
+  - Analysts gain more flexibility in managing workflows without escalating to administrators.
 
-    This widget can be configured to place button within any record's detail helping trigger playbooks at the click of a button.
+- **Source Control Settings Card**
 
-- **JSON to Grid** widget now has following capabilities:
+  * A new card under both *Production* and *Development* tabs consolidates management of exported content such as playbooks, module definitions, connectors, and related components.
+  * This streamlines updates and ensures critical components are version-controlled.
 
-    - Execute grid operations of **Production** or **Development** environment on **Playbook Execution Wizard** widget 
+- **Review and Apply Latest Content**
 
-    - Show grid columns even if there is no change request record present  
+  * A new button in both *Production* and *Development* environments triggers an **import wizard** that guides analysts through applying content changes.
+  * The wizard applies both *Production Content* and *Production Settings* safely, with a clear preview of what will change.
+  * A new **CR Status** column under issues shows the lifecycle state of each CR, improving at-a-glance tracking.
 
-    - Supported auto refresh of the Grid on create of the change request record 
+- **Improved PR Workflow**
 
-- **CICD Configuration Wizard** now offers **GitLab**, along with GitHub, as a source control platform and helps configure the selected source control with *Continuous Delivery* solution pack.
+  - When submitting a PR for review, analysts can now **select reviewers from a drop-down list of Git users** rather than typing usernames manually.
+  - Repositories and PR creation notifications now include **direct links** to the created repository or PR, reducing navigation overhead.
 
-## UI Enhancements
+---
 
-- Refreshed infographic on the **Setup** tab of the *Continuous Delivery* solution pack.
+## Bug Fixes
 
-- Refreshed infographic on the Continuous Delivery configuration wizard.
+- Fixed issues where:
 
-## Other Enhancements
+  - Multiple production environments were not handled correctly.
+  - Repository creation could fail if the repository already existed.
+  - Production/Development tabs appeared prematurely before source control setup.
+  - PR links were not visible in the UI when multiple issues were listed.
+  - Change Management tab visibility was inconsistent.
+  - Certain playbooks failed (e.g., create issue, submit for review).
```

Comment:
Error handling was improved in playbooks (CICD Taks) such as (e.g., create issue, submit for review).

Reply:
Added